packages/blade/src/components/Button/Button/Button.stories.tsx

```diff
@@ -148,6 +150,37 @@ const ButtonWithVariantTemplate: ComponentStory<typeof ButtonComponent> = ({
   );
 };
 
+const ButtonLoadingExample = (args: ButtonProps): React.ReactElement => {
```

Comment:
Can you move this after icon only stories like we discussed?

Reply:
Will do.